templates/partials/summary/summary.html

```diff
@@ -4,6 +4,8 @@
   {% from "components/summary/_macro.njk" import onsSummary %}
 
   {% set summaryGroups = [] %}
+  {% set summaryType = content.summary.summary_type %}
```

Comment:
I don't like that we set vars in camel case here when they are not DS elements, I mean all three. In other files we have things like:
- `{% set save_on_signout = true %}` in calculated summary html
- `{% set sign_out_url = content.sign_out_url %}`  and `{% set hide_sign_out_button = content.hide_sign_out_button %}` on thank you page
- `{% set page_title = _("Confirmation email sent") %}` on confirmation page

We should follow suit.

Reply:
Updated this 👍